changelog/fix16997.dd

```diff
@@ -0,0 +1,21 @@
+Preview switch fix16997 has now been set to true by default now
```

Comment:
The switch is `-preview=intpromote`, fix16997 is only an internal name

Reply:
done